mix.exs

```diff
@@ -88,7 +94,15 @@ defmodule Membrane.VideoCompositor.Mixfile do
       extras: ["README.md", "LICENSE"],
       formatters: ["html"],
       source_ref: "v#{@version}",
-      nest_modules_by_prefix: [Membrane.VideoCompositor]
+      nest_modules_by_prefix: [
+        Membrane.VideoCompositor,
+        Membrane.VideoCompositor.VideoTransformations
+      ],
+      groups_for_modules: [
+        "Video transformations": [
+          ~r/^Membrane\.VideoCompositor\.VideoTransformations($|\.)/
+        ]
+      ]
```

Comment:
It does not exatly fit into side bar  
![image](https://user-images.githubusercontent.com/34174337/213694987-51d25faf-23e4-4de5-afb6-365bf5c0fde6.png)

Reply:
That's true, however, I think it's not possible to fit them - at least either the header (`Membrane.VideoCompositor.VideoTr...`) is cut or the module names are. I think the solution would be to shorten these java-like names, for example by removing the `.VideoTransformations` part, though it's out of scope of this PR.